<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ML Lab  3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="graph">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Line settings</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Width:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="lineWidth">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Color:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="lineColor">
                     <item>
                      <property name="text">
                       <string>magenta</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>red</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>blue</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>green</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>cyan</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>yellow</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>black</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>white</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Style:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="lineStyle">
                     <item>
                      <property name="text">
                       <string>solid</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>dashed</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>dashdot</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>dotted</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_7">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Marker settings</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Count:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="markerCount">
                     <property name="maximum">
                      <number>200</number>
                     </property>
                     <property name="value">
                      <number>120</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Color:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="markerColor">
                     <item>
                      <property name="text">
                       <string>blue</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>red</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>green</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>cyan</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>magenta</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>yellow</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>black</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>white</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Style:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="markerStyle">
                     <item>
                      <property name="text">
                       <string>point</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>none</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>pixel</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>circle</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>octagon</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>square</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>triangle_down</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>triangle_up</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>star</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>plus_filled</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Function:Y(x) = cos(x ^ 2) / x, x = [1...10]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="show_plot">
              <property name="text">
               <string>Show Graphic</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Histogram</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Initial analysis for breast cancer data. Histogram for key features.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_777">
            <property name="text">
             <string>Labels color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="labels_color">
            <item>
             <property name="text">
              <string>red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cyan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>magenta</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>black</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>white</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="show_histogram">
            <property name="text">
             <string>Show Diagram</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
